src/gatling/scala/SingleLeaseMultipleChecks.scala

```diff
@@ -0,0 +1,46 @@
+import com.typesafe.config.{Config, ConfigFactory}
+import com.warrenstrange.googleauth.GoogleAuthenticator
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+import scala.concurrent.duration._
+
+class SingleLeaseMultipleChecks extends Simulation {
+  val config: Config = ConfigFactory.load()
+
+  val authenticator: GoogleAuthenticator = new GoogleAuthenticator()
+
+  val requestJwt = exec(
+    http("Lease token")
+      .post("/lease")
+      .formParam("microservice", config.getString("service.name"))
+      .formParam("oneTimePassword", authenticator.getTotpPassword(config.getString("service.pass")))
+      .check(bodyString.saveAs("jwt"))
+  )
+
+  val checkJwt = exec(
+    http("Check token")
+      .get("/details")
+      .header("Authorization", "Bearer ${jwt}")
+      .check(substring(config.getString("service.name")))
+  )
+
+  setUp(
+    scenario("Testing")
+      .exec(requestJwt)
+      .repeat(config.getInt("repeats")) {
+        exec(checkJwt)
+      }
+      .inject(
+        rampUsers(10) over (10 seconds),
```

Comment:
What's the purpose of this config? I guess we just need a ramp up to max users over N seconds and then keep running the simulation with max users for some time to see if it can handle the load. Probably with realistic pauses inside the scenario.

Reply:
Thanks Leszek, I am changing the plan.